playwrite-tests/web/src/main.rs

```diff
@@ -43,11 +48,16 @@ fn app(cx: Scope) -> Element {
             class: "eval-button",
             onclick: move |_| {
                 // Set the window title
-                let result = eval(r#"window.document.title = 'Hello from Dioxus Eval!';
-                return "returned eval value";"#.to_string());
-                if let Ok(serde_json::Value::String(string)) = result.get() {
-                    eval_result.set(string);
-                }
+                eval.run().unwrap();
+                let receiver = eval.receiver();
+                let setter = eval_result.setter();
+
+                wasm_bindgen_futures::spawn_local(async move {
```

Comment:
Could we just return an async block here or use cx.spawn instead of pulling in another library?

Reply:
Turns out we don't even need async.